XenAdmin/Dialogs/EnablePvsReadCachingDialog.cs

```diff
@@ -139,9 +139,7 @@ private AsyncAction GetAsyncActionForVm(VM vm, PVS_site siteSelected)
         /// <returns></returns>
         private bool PvsProxyAlreadyEnabled(VM vm)

```

Comment:
Here too I don't think we need to keep this method any longer, we can just keep it simple and write in line 121:
```
if (vm.PvsProxy != null)
    return null;
```